nrf_rpc/nrf_rpc.c

```diff
@@ -0,0 +1,786 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#define NRF_RPC_LOG_MODULE NRF_RPC
+#include <nrf_rpc_log.h>
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <string.h>
+
+#include "nrf_rpc.h"
+#include "nrf_rpc_tr.h"
+
+#define ERROR_FLAG_REMOTE 1
+#define ERROR_FLAG_SEND 2
+
+#define CMD_FLAG_WITH_RSP 0x10000
+
+enum {
+	/* Packet types with header variant 1:
+	 *      byte 0: bit 7: packet type, bits 6-0: group id,
```

Comment:
The docs indicate that group id is has a valid range of 0 - 127. Is that correct? Here it is mentioned that group id is only 6 bits

Reply:
Bit range from 6 to 0, so 7 bits. Do you think that this notation may be confusing? I can change it if yes.

Reply:
The header structure was redefined, so this comment in code does not exists any more.